include/deal.II/lac/scalapack.h

```diff
@@ -421,6 +421,34 @@ class ScaLAPACKMatrix : protected TransposeTable<NumberType>
   std::vector<NumberType> eigenpairs_symmetric_by_value(const std::pair<NumberType,NumberType> &value_limits,
                                                         const bool compute_eigenvectors);
 
+  /**
+   * Computing selected eigenvalues and, optionally, the eigenvectors of the real symmetric
+   * matrix $A \in \mathbb{R}^{M \times M}$ using the MRRR algorithm.
```

Comment:
in https://github.com/dealii/dealii/pull/5899#issuecomment-370182879 we agreed to use `\mathbf` throughout.